code/game/machinery/cryopod.dm

```diff
@@ -182,7 +182,6 @@ GLOBAL_LIST_INIT(frozen_items, list(SQUAD_MARINE_1 = list(), SQUAD_MARINE_2 = li
 /obj/structure/machinery/cryopod/Initialize()
 	. = ..()
 	announce = new /obj/item/device/radio/intercom(src)
-	flags_atom |= USES_HEARING
 
 /obj/structure/machinery/cryopod/Destroy()
 	QDEL_NULL(occupant)
```

Comment:
If there is an occupant, and this is destroyed, we will cause our reference in special_hearing_objects to go stale.

Also despawn_occupant below will leave this in the list incorrectly.